README.md

```diff
@@ -0,0 +1,494 @@
+like `totalLinesReady` and `bufferCount` help illustrate how much work is left to be done.  Any buffers that have been queued will
+be sent one after another, ignoring any flush timer.
+
+
+### Event: `'warn'`
+* [`<Object>`][]
+    * `message` [`<String>`][] - The warn message.  Depends on context.
+
+This is emitted when there is no log data provided to the `log` method, or when `removeMetaProperty` is called with an unrecognized key.  
+For those cases, additional properties (apart from `message`) are included:
+
+**Warnings during `log()`**
+
+* `statement` (Any) - If `log()` was called with a `null` string, or an invalid data type, this will contain the given log statement
+
+**Warnings during `removeMetaProperty`**
+
+* `key` [`<String>`][] - The key that was attempted to be removed, but did not exist
+
+## API
+
+### `createLogger(key[, options])`
+
+* `key` [`<String>`][] - Your [ingestion key](https://docs.logdna.com/docs/ingestion-key)
+* `options` [`<Object>`][]
+    * `level` [`<String>`][] - [Level](#supported-log-levels) to be used if not specified elsewhere. **Default:** `INFO`
+    * `tags` [`<Array>`][] | [`<String>`][] - Tags to be added to each message
+    * `meta` [`<Object>`][] - Global metadata.  Added to each message, unless overridden.
+    * `timeout` [`<Number>`][] - Msecond timeout for each HTTP request. **Default:** `5000`ms. **Max:** `300000`ms
+    * `hostname` [`<String>`][] - Hostname for each HTTP request.
+    * `mac` [`<String>`][] - MAC address for each HTTP request.
+    * `ip` [`<String>`][] - IPv4 or IPv6 address for each HTTP request.
+    * `url` [`<String>`][] - URL of the logging server.  **Default:** `https://logs.logdna.com/logs/ingest`
+    * `flushLimit` [`<Number>`][] - Maximum total line lengths before a `flush` is forced. **Default:** `5000000`
+    * `flushIntervalMs` [`<Number>`][] - Mseconds to wait before sending the buffer. **Default:** `250`ms
+    * `shimProperties` [`<Array>`][] - List of dynamic `options` keys to look for when calling `log()`
+    * `indexMeta` [`<Boolean>`][] - Controls whether or not `meta` data for each message is searchable. **Default:** `false`
+    * `app` [`<String>`][] - Arbitrary app name for labeling each message. **Default:** `default`
+    * `env` [`<String>`][] - An environment label attached to each message
+    * `baseBackoffMs` [`<Number>`][] - Minimum exponential backoff time in milliseconds. **Default:** `3000`ms
+    * `maxBackoffMs` [`<Number>`][] - Maximum exponential backoff time in milliseconds. **Default:** `30000`ms
+    * `withCredentials` [`<Boolean>`][] - Passed to the request library in order to make CORS requests. **Default:** `false`
+* Throws: [`<TypeError>`][] | [`<TypeError>`][] | [`<Error>`][]
+* Returns: `Logger`
+
+Returns a logging instance to use.  `flushLimit` and `flushIntervalMs` control when the buffer is sent to LogDNA, whichever comes first.  
```

Comment:
Yeah, this one was tough, especially since the timer isn't really a `value` for anything.  You read it right though.  Either the timer fires, or the limit is reached.  I re-worked the sentence to be more clear (I think) :]